src/pictures/HomePagePics.js

```diff
@@ -0,0 +1,26 @@
+import DannyDevitoLegend from './Danny_Devito_themanthemyththelegend.jpg'
+import DannyBuscemi from "./DannyBuscemi.jpg"
+import Firefighter from './firefighter.jpg'
+import SteveBuscemiHead from './SteveBuscemiHead.jpg'
+import SteveDevito from './SteveDevito.jpg'
+import SteveHotty from './SteveHotty.jpg'
+
+
+
+export const HomePagePics = [
```

Comment:
It would be better to pair the pic with the caption instead of keeping them separate. 

```suggestion
export const HomePagePics = [
  {
    "picture": DannyDevitoLegend,
    "caption": "Danny Devito - The Man, The Myth, The Legend"
  }
]
```

Reply:
Oh I thought I was doing that by putting them in the same object- what did you mean?